renderdoc/driver/d3d12/d3d12_sdk_select.cpp

```diff
@@ -288,12 +443,18 @@ void D3D12_PrepareReplaySDKVersion(UINT SDKVersion, bytebuf d3d12core_file,
         FileIO::fwrite(d3d12core_file.data(), 1, d3d12core_file.size(), f);
         FileIO::fclose(f);
 
-        D3D12Core_Override_Path = get_dirname(filename);
-        D3D12Core_Temp_Path = get_dirname(filename);
+        if(!IsSignedByMicrosoft(filename))
+        {
+          FileIO::Delete(filename);
+          RDCERR(
```

Comment:
Can you add a user setting (see `RDOC_CONFIG` at the top of the file) which is default off, that allows these checks to be demoted to warnings and then load the DLL anyway? That way if someone has problems it can be easily tested to see if this check is the cause.